packages/tradershub/src/components/UIProvider/UIProvider.tsx

```diff
@@ -3,7 +3,7 @@ import React, { createContext, useCallback, useContext, useMemo, useState } from
 type TUIState = {
     // Add other UI states here
     accountType?: string;
-    isSignupWizardOpen?: boolean;
+    isSignupWizardOpen: boolean;
```

Comment:
@shahzaib-deriv If we end up with reactmodal for all modals, we can keep modalProvider for hiding/showing all the modals. This way we can separate the responsibility of each provider, CFD, UI, Modal
What do you think?